.github/workflows/test.yml

```diff
@@ -0,0 +1,33 @@
+---
+name: Tests
+on: [ push, pull_request ]
+jobs:
+  test:
+    name: test (Ruby ${{ matrix.ruby }})
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: [ '2.5', '2.6', '2.7', '3.0', '3.1' ]
```

Comment:
The project needs a fix for Ruby 3.2. I'll investigate this in a subsequent pull request.